### PR TITLE
add : added badge .html and badge.css

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
1. Added badge Component
2. Added documentation of badge component

image

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+.badge-container {
+    display: flex;
+    gap: 1rem;
+    align-items: baseline;
+    border:1px solid var(--grey);
+    padding:25px;
+    width:80%;
+    background-color: rgb(248, 244, 244);
+    margin-bottom:1rem;
```
Comment (Ankur9669) [inline components/badge/badge.css:22]: Use css variables for bg color

File: components/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    padding: 5px;
+    top: 0px;
+    right: 0px;
+    border: 2px solid var(--white);
+  }
+  
+  .bell-badge {
+    height: 30px;
+    width: 3rem;
+    left: 2rem;
```
Comment (Ankur9669) [inline components/badge/badge.css:47]: Either use px or rems, here you have used both

File: components/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+  
+  
+  .avatar-xl {
+    bottom: 70px;
+    right: 5px;
+  }
+  
+  .avatar-l {
+    bottom: 62.5rem;
+    right: 0px;
```
Comment (Ankur9669) [inline components/badge/badge.css]: Here also you are using combination of rem and px

File: components/badge/badge.css [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+    height:5rem;
+    width: 5rem;
+    border-radius: 50%;
+    background-color:var(--orange);
+}
+
+.avatar-m {
+    height: 60px;
+    width: 60px;
+    border-radius: 50%;
```
Comment (Ankur9669) [inline components/badge/badge.css:133]: .avatar-l {
    height:5rem;
    width: 5rem;
    border-radius: 50%;
    background-color:var(--orange);
}

See here you have used rems for width and height
So use rems in other classes also for consistency

File: components/badge/badge.css [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+
+.avatar-s {
+    height: 40px;
+    width: 40px;
+    border-radius: 50%;
+    background-color: var(--blueviolet);
+}
+
+@media screen and (max-width:658px){
+    .badge-container {
```
Comment (Ankur9669) [inline components/badge/badge.css:145]: Use @media only screen and max width----
Here you have missed **only**

File: components/badge/badge.html [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+                style=" height: 200px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
```
Comment (Ankur9669) [inline components/badge/badge.html:84]: Could remove inline styling

File: components/badge/badge.html [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+
+            <div class="avatar-badge-parent"><img class="avatar avatar-xl" src="/assets/avatar.jpg"><span
+                    class="avatar-badge online badge-xl"></span></img></div>
+
+            <div class="avatar-badge-parent"> <img class="avatar avatar-l " src="/assets/avatar.jpg"><span
+                    class="avatar-badge offline badge-l"></span></div>
+
+            <div class="avatar-badge-parent"> <img class="avatar avatar-m" src="/assets/avatar.jpg"><span
+                    class="avatar-badge busy badge-m"></span></div>
+
```
Comment (Ankur9669) [inline components/badge/badge.html:102]: img tags do not have alt tag

File: components/badge/badge.html [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+            <li><a href="/components/html/nav.html">Navigation-Desktop</a></li>
+            <li><a href="/components/html/modal.html">Modal</a></li>
+            <li><a href="/components/html/rating.html">Rating</a></li>
+            <li><a href="/components/html/toast.html">Toast/Snackbar</a></li>
+            <li><a href="/components/html/card.html">Simplified Grid</a></li>
+        </ul>
+    </div>
+
+    <div class="Component-container">
+
```
Comment (Ankur9669) [inline components/badge/badge.html:49]: could use main instead of div here

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
Most of the coding conventions look good to  me, these are some changes that you can do.
I have not reviewed your UI as preview is not available, so i would be waiting for you preview link

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Screenshot (34)

your badge text are not align centered, and some badges are not round, also badges positioning is not looking good, may be move upward or right little bit

File: components/badge/badge.html [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/main.css">
+    <link rel="stylesheet" href="/doc.css">
+    <link rel="stylesheet" href="/components/badge/badge.css">
+    <script src="https://kit.fontawesome.com/9c98d67da5.js" crossorigin="anonymous"></script>
+    <script src="https://kit.fontawesome.com/9c98d67da5.js" crossorigin="anonymous"></script>
```
Comment (gauravsinhaweb) [inline components/badge/badge.html]: script is repeating here.

File: components/badge/badge.html [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+        </ul>
+    </div>
+
+    <div class="Component-container">
+
+
+        <!---------badge componenet-------->
+        <div class="text mt4 " id="badge">
+
+            <h2 class="pb1 h1 comp-title">Badges</h2><br>
```
Comment (gauravsinhaweb) [inline components/badge/badge.html:54]: avoid using br tag for breaking line for element tag instead use div. <br/> is good to use inside paragraph or text.

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]
Good to merge after changes. Nice work!

File: components/badge/badge.css (hunk 1 above)
Comment (JuvitaSaini09) [reply components/badge/badge.css:22]: I have made the change ,thankyou for the review

File: components/badge/badge.css (hunk 2 above)
Comment (JuvitaSaini09) [reply components/badge/badge.css:47]: i have fixed it

File: components/badge/badge.css (hunk 3 above)
Comment (JuvitaSaini09) [reply components/badge/badge.css]: i have fixed it,thankyou

File: components/badge/badge.css (hunk 5 above)
Comment (JuvitaSaini09) [reply components/badge/badge.css:145]: i have fixed it,thankyou for the review

File: components/badge/badge.html (hunk 7 above)
Comment (JuvitaSaini09) [reply components/badge/badge.html:102]: i have added the alt,thnks for review

File: components/badge/badge.html (hunk 8 above)
Comment (JuvitaSaini09) [reply components/badge/badge.html:49]: i have changed div to main ,thankyou for review

File: components/badge/badge.html (hunk 9 above)
Comment (JuvitaSaini09) [reply components/badge/badge.html]: i have removed it,thnks for reviewing

File: components/badge/badge.html (hunk 10 above)
Comment (JuvitaSaini09) [reply components/badge/badge.html:54]: ok i will take care of it

**Review by Ankur9669 — APPROVED** [review]
Nice work!!. Looks good to merge

**Review by gauravsinhaweb — APPROVED** [review]

**Pull request merged**